src/clj/commiteth/eth/multisig_wallet.clj

```diff
@@ -191,30 +191,6 @@
                                  [tla (token-balance bounty-addr tla)]))) addrs)))
       {})))
 
-(defn transfer-tokens
-  "Transfer mount of given ERC20 token from from-addr to
-  to-addr. Connected geth needs to have keys for the account and
-  passphrase needs to be supplied. Returns transaction ID."
-  [from-addr from-passphrase token to-addr amount]
-  (let [token-addr (get-token-address token)]
-    (eth/execute-using-addr from-addr
-                            from-passphrase
-                            token-addr
-                            (method-ids :transfer)
-                            to-addr
-                            amount)))
-
-(defn get-owners
-  "Return vector of multisig owner addresses."
-  [bounty-addr]
-  (let [bounty-contract (load-bounty-contract bounty-addr)
-        owner-addresses (-> bounty-contract
-                            (.getOwners)
-                            .get)]
-    (if owner-addresses
-      (mapv #(.toString %) (.getValue owner-addresses))
-      [])))
-
```

Comment:
This and a bunch of other stuff has been deleted — is it simply not needed any more at all or is this in relation to the change introduced by this PR? If it's just dead code it would be nicer to review in separate PRs but fine for now.

Reply:
I was prompted to check those functions because they would've been impacted by the PR. Turned out they are not used anywhere.

Reply:
Ok nice. Deleting code == good.